app/main.py

```diff
@@ -1,3 +1,66 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    Variables = "Distance", int, float
```

Comment:
The `Variables` type alias is not correctly defined. It should be a type hint using `Union` or `TypeVar` to specify the allowed types, such as `Union[Distance, int, float]`.